resolwe/flow/executors/prepare.py

```diff
@@ -51,6 +51,8 @@ def extend_settings(self, data_id, files, secrets):
         )
         files[ExecutorFiles.DATA] = model_to_dict(data)
         files[ExecutorFiles.DATA_LOCATION] = model_to_dict(data.location)
+        # Add subpath for backward compatibility since it is not a field in this model anymore
```

Comment:
Where do we need this?

Reply:
It is used in worker (run.py in docker for example).

Later the key was renamed from ExecutorFiles.DATA_LOCATION to ExecutorFiles.STORAGE_LOCATION.